vumi/middleware/logging.py

```diff
@@ -1,9 +1,24 @@
 # -*- test-case-name: vumi.middleware.tests.test_logging -*-
 
+from confmodel import Config
+from confmodel.fields import ConfigText
+
 from vumi.middleware import BaseMiddleware
 from vumi import log
 
 
+class LoggingMiddlewareConfig(Config):
```

Comment:
Yes, I made `BaseMiddlewareConfig` after I made all these configs, so I'll change them to extend that instead.